### ISSUE TITLE
Training loss collapse 

### ISSUE THREAD

**Issue by aisagarw** [taoyang1122/pytorch-SimSiam#9]
I am trying to pretrain the SimSiam model on mscoco dataset.. but the loss collapses to -1 very quickly.. What are the possible reasons behind and some suggestions to solve the same?
